benches/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2018"
 
 [dependencies]
 tokio = { version = "0.3.0", path = "../tokio", features = ["full"] }
+tokio-util = { version = "0.4.0", path = "../tokio-util", features = ["full"] }
```

Comment:
is it possible to move this to `dev-dependencies`?

Reply:
Good point. I've updated the code.